pkgs/development/python-modules/detectron2/default.nix

```diff
@@ -79,14 +83,27 @@ buildPythonPackage {
     iopath
     omegaconf
     hydra-core
-    black
```

Comment:
There seems to be a method that uses black.
https://github.com/facebookresearch/detectron2/blob/47d8ed7a1bd999719825f2e8a11d5b2828bc02c4/detectron2/config/lazy.py#L390